### ISSUE TITLE
Permission issues . "no event found until midnight" despite "Full Access" to Calendar  

### ISSUE THREAD

**Issue by oschrenk** [0ihsan/mcal#6]
Not a bug report, more an observation and  comment if other people also run into this issue.

TL;DR

The terminal requires "Full Access" to  Calendar AND access to "Reminders"

---

Thank you for building the software. It scratches a very particular itch.

An issue I encountered is that enabling "Full Access" to the Calendar was not enough.

The app always reported that there were no events

```
./mcal next
no event found until midnight
```

iCalBuddy also reported that there were no events today:

```
icalBuddy -f -sd eventsToday
error: No calendars.
```

This is despite granting Full Access to the Terminal (Alacritty in my case).

<img width="476" alt="Screenshot 2024-06-25 at 19 24 44" src="https://github.com/0ihsan/mcal/assets/52898/1360a9fd-c318-4846-80c1-b7165508c0ca">

Only after ALSO granting access to "Reminders"

<img width="480" alt="Screenshot 2024-06-25 at 19 18 59" src="https://github.com/0ihsan/mcal/assets/52898/7015c6c1-4736-4832-aae4-009f3cb2d80b">

Both apps, mcal and iCalBuddy were able to find events.

It's funny that

https://github.com/0ihsan/mcal/blob/ee06a64690c0da03a1aef14ea43932ed075e9539/mcal.swift#L44C1-L47C157

Actually, does the right check (to my eyes), but macOS still only allows to get calendar entries if you also grant access to Reminders. 

macOS 14.5 (23F79)

**Comment by arlyle** [comment]
thank you. apple integrated reminders(w/dates) to show in calendar on macos 15.0 sequoia. not sure why you had encountered this in macos 14.5 though...
![cr](https://github.com/0ihsan/mcal/assets/161149446/b18deaa6-399a-4285-9837-cde672b1d554)

**Comment by 0ihsan** [comment]
thank you. apple integrated reminders(w/dates) to show in calendar on macos 15.0 sequoia. not sure why you had encountered this in macos 14.5 though...
![cr](https://github.com/0ihsan/mcal/assets/24369769/3130ab2c-f9a7-4f70-bde6-07049794828f)
